README.md

```diff
@@ -65,50 +72,111 @@ REPOSITORY          TAG                 IMAGE ID            CREATED
+password.
+
+> _nota bene_: you'll probably be asked to accept the connection even if it is
+> insecure (SSL certificate issue). Please do so.
+
+And finally you must also login from the CLI to be allowed to perform requests
+on the OpenShift cluster:
 
 ```bash
-$ bin/ansible-playbook load_fixtures.yml
+$ oc login --insecure-skip-tls-verify=true -u developer -p developer "${K8S_AUTH_HOST}"
 ```
 
-The following users will be created:
+As we are gentle people, we also provide a shortcut to automate running a new
+cluster and login to it (GNU/Linux only for now):
+
+```
+$ bin/dev
+```
+
+Please, note that **you do not need to run** `bin/dev` if you have already
+started a local cluster and logged in with `oc`.
+
+### Deploy!
+
+Now that you have a working OpenShift cluster, let's have fun (sic!) by creating
+a project for a customer in a particular environment with a new helper:
+
+_nota bene_: when running this command, you'll be asked for a **vault
+password**. The default value for this demo is: `arnold`.
+
+```bash
+$ bin/bootstrap
+```
+
+Tadaaa! Arnold has created a new OpenShift project called `patient0-development`
+with a collection of services up and running.
+
+When `edxapp`'s first deployment has completed successfully, we invite you to
+test the lms or studio application with the following credentials:
 
 | username | password | email               | is staff | is superuser |
-|---------:|:--------:|:-------------------:|:--------:|:------------:|
-| student  | student  | student@example.com | no       | no           |
-| teacher  | teacher  | teacher@example.com | no       | no           |
-| staff    | staff    | staff@example.com   | yes      | no           |
-| admin    | admin    | admin@example.com   | yes      | yes          |
+| :------- | :------- | :------------------ | :------: | :----------: |
+| student  | student  | student@example.com |    no    |      no      |
+| teacher  | teacher  | teacher@example.com |    no    |      no      |
+| staff    | staff    | staff@example.com   |   yes    |      no      |
+| admin    | admin    | admin@example.com   |   yes    |     yes      |
+
+_nota bene_: you will find URLs of the studio or lms services from the web
+console (Application > Routes). You should see a page like the above screenshot.
+
+![OpenShift routes](./docs/images/os-web-console-routes.png)
 
+### Going further
 
-You could also try deploying the same services for another customer and/or environment by
-overriding the default Ansible variables:
+You can deploy the same services for another customer and/or environment by
+overriding the default Ansible variables (`customer` and `env_type`
+respectively):
 
 ```bash
 $ bin/bootstrap -e "env_type=staging customer=campus"
 ```
 
-If you want to run a new deployment for this project, there is also a helper for
-that:
+If you want to run a new deployment for the default project / environment, there
+is also a helper for that:
```

Comment:
Not very helpful instruction. I have the feeling that before you can run this command, you have to understand a bit more than that... and it must comme with instructions about the "switch" command. Should we leave this for the detailed documentation and remove it from the README?